src/crate/crash/repl.py

```diff
@@ -184,19 +233,21 @@ def __call__(self, buffer):
             return
 
         current_line = buffer.document.text
-        cursor_position = buffer.document.cursor_position
 
-        if self.last_changed and self.is_prefix(current_line[:cursor_position].lower(), self.last_changed.lower()):
-            diff = len(self.last_changed) - len(current_line)
-            current_line = self.last_changed + current_line[diff:]
+        if current_line[0] != '\\':
```

Comment:
code style: make as little indentation (nested if blocks) as possible

```py
if current_line.startswith('\\'):
    return
```

then you can leave the indentation of the rest as is